src/main/java/gtPlusPlus/xmod/gregtech/loaders/RecipeGen_BlastSmelterGT_GTNH.java

```diff
@@ -158,20 +158,40 @@ public synchronized static boolean generateGTNHBlastSmelterRecipesFromEBFList()
 					}
 					//If this recipe is enabled and we have a valid molten fluidstack, let's try add this recipe.
 					if (enabled && isValid(inputs, outputs, inputsF, mMoltenStack)) {						
+						// Boolean to decide whether or not to create a new circuit later
+						boolean circuitFound = false;
+
 						//Build correct input stack
 						ArrayList<ItemStack> aTempList = new ArrayList<ItemStack>();
 						for (ItemStack aPossibleCircuit : inputs) {
 							if (!ItemUtils.isControlCircuit(aPossibleCircuit)) {
```

Comment:
```
	if (ItemUtils.isControlCircuit(aPossibleCircuit)) {
		circuitFound = true;
	}
	aTempList.add(aPossibleCircuit);
```

Probably a tad cleaner.

Reply:
Right, I forgot to clean this up after I checked it works. I also change the "aPossibleCircuit" variable name to just recipeItem because it confused me at first, I don't like it.